lib/puppet/provider/package/portage.rb

```diff
@@ -119,6 +119,8 @@ def qatom
           # some fields can be empty or (null) (if we are not passed a category in the package name for instance)
           if value == '(null)'
             package_info[field] = nil
+          elsif value == '<unset>'
```

Comment:
All of these cases end up containing the same code. I think I'd rather see them all or'd together instead of being an elsif chain like this?

Reply:
I could see that for null and unset valuse, but including a bunch of conditionals on one line makes the code harder to read imo.